config/v1/types/disk.go

```diff
@@ -31,25 +30,25 @@ func (n Disk) Validate() report.Report {
 	if len(n.Device) == 0 {
 		r.Add(report.Entry{
 			Kind:    report.EntryError,
-			Message: "disk device is required",
+			Message: errors.ErrDiskDeviceRequired.Error(),
 		})
 	}
 	if n.partitionNumbersCollide() {
 		r.Add(report.Entry{
 			Kind:    report.EntryError,
-			Message: fmt.Sprintf("disk %q: partition numbers collide", n.Device),
+			Message: errors.ErrPartitionNumbersCollide.Error(),
```

Comment:
To prevent the loss of information would it make sense to add a generator function for these error types that accepts parameters for formatting?

Reply:
I think it's ok, since we're telling them where in the config the error is.

Reply:
Yup, I removed the context included in these errors since the validation logic provides line/col information about errors, and with my other PR will even print out the line these errors come from.

Reply:
I did leave some `fmt.Errorf` errors in the source though since some of them were providing more information about the error, instead of more context about where the error came from.